src/containers/linking/LinkingReducer.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * @flow
+ */
+
+import { Map, fromJS } from 'immutable';
+import { DataApiActions } from 'lattice-sagas';
+import { RequestStates } from 'redux-reqseq';
+import type { SequenceAction } from 'redux-reqseq';
+
+import {
+  SEARCH_LINKED_ENTITY_SETS,
+  searchLinkedEntitySets,
+} from './LinkingActions';
+
+const {
+  GET_LINKED_ENTITY_SET_BREAKDOWN,
+  getLinkedEntitySetBreakdown,
+} = DataApiActions;
+
+const INITIAL_STATE :Map<*, *> = fromJS({
+  [GET_LINKED_ENTITY_SET_BREAKDOWN]: {
+    requestState: RequestStates.STANDBY,
+  },
+  [SEARCH_LINKED_ENTITY_SETS]: {
+    requestState: RequestStates.STANDBY,
+  },
+  links: Map(),
+  searchResults: Map(),
+});
+
+export default function reducer(state :Map<*, *> = INITIAL_STATE, action :Object) {
+
+  switch (action.type) {
+
+    case getLinkedEntitySetBreakdown.case(action.type): {
+      return getLinkedEntitySetBreakdown.reducer(state, action, {
+        REQUEST: () => {
+          const seqAction :SequenceAction = action;
+          return state
+            .setIn([GET_LINKED_ENTITY_SET_BREAKDOWN, 'requestState'], RequestStates.PENDING)
+            .setIn([GET_LINKED_ENTITY_SET_BREAKDOWN, seqAction.id], seqAction);
```

Comment:
I suspect I should probably get the BHR to follow this pattern if everyone else is also storing the ids to active actions.

Reply:
It's only necessary if you need to retrieve the original trigger action object.